frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.js

```diff
@@ -268,6 +280,40 @@ class TracingActionsView extends React.PureComponent<Props, State> {
     location.href = `/annotations/Explorational/${newAnnotation.id}`;
   };
 
+  handleCopySandboxToAccount = async () => {
+    const { tracing: sandboxTracing, dataset } = Store.getState();
```

Comment:
I agree and I just refactored the code accordingly only to notice that it doesn't really work, because as part of the api function `restart` which is used in this method, all the sagas are restarted as well :see_no_evil:

Reply:
Oh no :see_no_evil: Then, let's forget that this happened :D